src/test/java/com/robinpowered/sdk/model/BasicIdentifierTest.java

```diff
@@ -3,11 +3,11 @@
 import nl.jqno.equalsverifier.EqualsVerifier;
 import org.junit.Test;
 
-public class IdentifierTest {
+public class BasicIdentifierTest {
 
     @Test
     public void testEqualsAndHashcode() {
-        EqualsVerifier.forClass(Identifier.class)
+        EqualsVerifier.forClass(BasicIdentifier.class)
                 .usingGetClass()
                 .verify();
     }
```

Comment:
Added in https://github.com/robinpowered/robin-java-sdk/pull/17/commits/b8479c47b9c80519d226ce1430b3fe8101da7338